setup.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2013, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from setuptools import setup
+
+__version__ = "1.0.2"
```

Comment:
Not sure, I think this is the default value we are using on all the plugins, @josenavas, what do you think?